### ISSUE TITLE
Classifying videos

### ISSUE THREAD

**Issue by kc9jud** [marcelklehr/recognize#157]
It seems useful (and not too difficult) to tag videos by extracting some frames and running the normal networks on them. One option would be something like this: https://jdhao.github.io/2021/12/25/ffmpeg-extract-key-frame-video/

I might work on a PR if there's sufficient interest.

**Comment by bonswouar** [comment]
That would be great to be able to recognize videos too, and that approach shouldn't be too hard to implement & pretty quick to execute, +1 for me! ;)

Although we have to agree on some specifications, mainly how to chose keyframes.
Something like [that](https://www.sciencedirect.com/science/article/abs/pii/S0031320321004532) would be the best, but I believe there is no available free model of this kind (yet).

Meanwhile we could go on some arbitrary timerange (and maybe adapt it depending on the full length of the video), and exclude too similar / too dark images. For all that we could quite easily adapt this from MediaDC plugin: https://github.com/andrey18106/mediadc/blob/main/lib/Service/python/dc_videos.py

@marcelklehr What do you think?

**Comment by marcelklehr** [comment]
Good idea, indeed. My first intuition would be to classify a frame every X seconds and accumulate all the resulting labels.

**Comment by marcelklehr** [comment]
Reopening to track face and imagenet recognition for videos
